fastchat/model/model_chatglm.py

```diff
@@ -37,6 +37,34 @@ def process_response(response):
     return response
 
 
+def apply_stopping_string(reply, stop_strings):
+    if isinstance(stop_strings, str):
+        stop_strings = [stop_strings]
+
+    stop_found = False
+
+    for string in stop_strings[:4]:
```

Comment:
Why do you use '4' here?

Reply:
I referred to OpenAI's [API reference](https://platform.openai.com/docs/api-reference/chat/create#chat-create-stop), where the description of the stop parameter is 'Up to 4 sequences where the API will stop generating further tokens.' I believe this is intended to deter malicious cyberattacks.

Reply:
We do not need to follow this restriction in this low-level utility function.
If needed, we can apply it on a higher level (e.g., API server)